Makefile

```diff
@@ -345,6 +346,10 @@ gatewayapitests:
 int_test:
```

Comment:
Can we add `eps_enabled` as part of `int_test` target

Reply:
Instead of adding eps_enabled to int_test, we can run complete int_test with endpoint slice enabled